tests/src/Functional/OgMultipleSelectionWidgetAutoCompleteTest.php

```diff
@@ -0,0 +1,199 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\og\Functional;
+
+use Drupal\node\Entity\Node;
+use Drupal\node\Entity\NodeType;
+use Drupal\og\Entity\OgRole;
+use Drupal\og\Og;
+use Drupal\og\OgGroupAudienceHelper;
+use Drupal\Tests\node\Traits\ContentTypeCreationTrait;
+use Drupal\Tests\BrowserTestBase;
+
+/**
+ * Testing the audience field with multiple auto select widgets.
+ *
+ * @group og
+ */
+class OgMultipleSelectionWidgetAutoCompleteTest extends BrowserTestBase {
+
+  use ContentTypeCreationTrait;
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = ['node', 'og'];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected $defaultTheme = 'stark';
+
+  /**
+   * A group node for user 1.
+   *
+   * @var \Drupal\node\Entity\Node
+   */
+  protected $group;
+
+  /**
+   * Group owner.
+   *
+   * @var \Drupal\user\Entity\User
+   */
+  protected $user;
+
+  /**
+   * OG role manager service.
+   *
+   * @var \Drupal\og\OgRoleManagerInterface
+   */
+  protected $roleManager;
+
+  /**
+   * The role of the user.
+   *
+   * @var \Drupal\og\OgRoleInterface
+   */
+  protected $role;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function setUp() {
+    parent::setUp();
+
+    // Create group node types.
+    $this->createContentType(['type' => 'group_type']);
+    Og::addGroup('node', 'group_type');
+
+    NodeType::create(['type' => 'group_content'])->save();
+
+    // Set the field widget as autocomplete..
+    $settings = [
+      'form_display' => [
+        'type' => 'entity_reference_autocomplete',
+      ],
+    ];
+    Og::createField(OgGroupAudienceHelper::DEFAULT_FIELD, 'node', 'group_content', $settings);
+
+    // Add another field.
+    $settings += [
+      'field_name' => 'override_name',
+      'field_config' => ['label' => 'Second group reference'],
+    ];
+    Og::createField(OgGroupAudienceHelper::DEFAULT_FIELD, 'node', 'group_content', $settings);
+
+    // Create users.
+    $this->user = $this->drupalCreateUser();
+
+    // Create groups.
+    $this->group = Node::create([
+      'type' => 'group_type',
+      'title' => 'group1',
+      'uid' => $this->user->id(),
+    ]);
+    $this->group->save();
+
+    // Adding to the member role the appropriate permission.
+    $this->role = OgRole::create();
+    $this->role
+      ->setName('content_editor')
+      ->setLabel('Content group editor')
+      ->setGroupType('node')
+      ->setGroupBundle('group_type')
+      ->grantPermission('create group_content content')
+      ->save();
+
+    // Create a role.
+    $role_storage = $this->container->get('entity_type.manager')->getStorage('user_role');
+    $role_storage->create([
+      'id' => 'dummy_role',
+      'permissions' => [
+        'create group_content content',
+        'edit own group_content content',
+        'edit any group_content content',
+      ],
```

Comment:
Drupal\Tests\og\Functional\OgMultipleSelectionWidgetAutoCompleteTest::testAutoCompleteForNonGroupMember
Role label is expected to be a string. in /home/travis/build/Gizra/drupal/core/modules/user/user.module:851

```suggestion
      ],
      'label' => 'Dummy role',
```